tests/helper_test_functions.py

```diff
@@ -0,0 +1,140 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.webelement import WebElement
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.support.wait import WebDriverWait
+
+from . import settings
+
+TIMEOUT_BUTTON = 2
+TIMEOUT_TEXTFIELD = 2
+
+ID_USERNAME_ELEMENT = "username_textfield"
+ID_PASSWORD_ELEMENT = "password_textfield"
+
+
+# The name used for the username text field
+# The id used for the login button
+ID_LOGIN_BUTTON_ELEMENT = "login_button"
+
+# The id used for the logout button in the home page
+ID_LOGOUT_ELEMENT = "logout_element"
+
+
+@pytest.fixture(scope="class")
+def browser_driver(request: FixtureRequest):
+    """Create the browser driver with the right request.
+
+    Creates the browser driver for uses in diffrent test.
+
+    Parameters:
+        request (FixtureRequest): For the config given by
+        the users.
+    """
+    driver: webdriver
+    match request.config.option.browser:
+        case "chrome":
+            options = webdriver.ChromeOptions()
+            if request.config.option.head == "1":
+                options.add_argument("--headless")
+            driver = webdriver.Chrome(options=options)
+        case "edge":
+            options = webdriver.EdgeOptions()
+            if request.config.option.head == "1":
+                options.add_argument("--headless")
+            driver = webdriver.Edge(options=options)
+        case "chromium":
+            options = webdriver.ChromeOptions()
+            if request.config.option.head == "1":
+                options.add_argument("--headless")
+            driver = webdriver.ChromiumEdge(options=options)
+        case _:
+            options = webdriver.FirefoxOptions()
+            print(request.config.option.headless)
+            if request.config.option.head == "1":
+                options.add_argument("--headless")
+            driver = webdriver.Firefox(options=options)
+    yield driver
+    driver.close()
+
+
+def is_in_login_screen(driver: webdriver) -> None:
+    """Check if the driver is currently at the login screen.
+
+    This functions checks if the driver is in the login screen
+    by checking if specific exist in the login screen.
+
+    Parameters:
+        driver (webdriver): The webdriver that would be checked.
+    """
+    msg = "It is not in the login screen"
+    assert get_element_by_id(ID_USERNAME_ELEMENT, driver) is not None, msg
+    assert get_element_by_id(ID_PASSWORD_ELEMENT, driver) is not None, msg
+
+
+def is_in_home_page(driver: webdriver) -> None:
+    """Check if the driver is currently at the home page.
+
+    This functions checks if the driver is in the home page
+    by checking if specific exist in the login screen.
+
+    Parameters:
+        driver (webdriver): The webdriver that would be checked.
+    """
+    assert get_element_by_id(ID_LOGOUT_ELEMENT, driver) is not None
+
+
+def try_login(username: str, password: str, driver: webdriver) -> None:
```

Comment:
In my opinion, `driver` should be the first argument, as it would be more consistent with other functions. Also applies to `get_element_by_id`.